packages/@lwc/wire-service/src/__tests__/wiring.spec.ts

```diff
@@ -14,7 +15,7 @@ import {
     CONTEXT_UPDATED,
     CONFIG,
 } from '../constants';
-import { Element, ElementDef, WireDef } from '../engine';
+import { LightningElement, ElementDef, WireDef } from '../engine';
```

Comment:
should the 'engine' be renamed to 'lwc'?

Reply:
engine is a local file that reexport stuff from lwc.